go.mod

```diff
@@ -23,7 +23,7 @@ require (
 	github.com/cosmos/cosmos-sdk v0.50.7
 	github.com/cosmos/gogoproto v1.7.0
 	github.com/decred/dcrd/dcrec/secp256k1/v4 v4.3.0
-	github.com/ethereum/go-ethereum v1.14.8
```

Comment:
Do we also need to rollback geth version?

Reply:
Yes, because of the compatibility issue of btcec. 
cometbft v0.38.9 uses v2.2.0 of btcec, but go-ethereum v1.14.8 uses v2.3.4 of btcec, which is not backward compatible.